app/src/main/res/navigation/nav_graph.xml

```diff
@@ -255,7 +255,6 @@
     <fragment
         android:id="@+id/bookmarkFragment"
         android:name="org.mozilla.fenix.library.bookmarks.BookmarkFragment"
-        android:label="@string/library_bookmarks"
```

Comment:
I'd prefer to change this to `tools:text` so it still appears in the Preview pane.

Reply:
Oops, this is actually the navigation graph. My bad!